clang/test/CodeGen/cheri/cheri-addrof-member.c

```diff
@@ -1,40 +1,41 @@
-// RUN: %cheri_cc1 %s -emit-llvm -o - | FileCheck %s
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
+// RUN: %cheri_cc1 -disable-O0-optnone -emit-llvm %s -o - \
```

Comment:
I feel like clang should really have something like a -Otest output flag to emit sane IR...

Reply:
I want -Oepsilon because it makes reading assembly so much saner, too... maybe one day I'll write that patch :)